src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Migrations/2.sql

```diff
@@ -716,6 +773,27 @@ AS
 
     SELECT @newWatermark
 
+    -- Insert metadata into custom tag tables
+    INSERT INTO dbo.CustomTagString
```

Comment:
How does this work for tags at study or series level. They will be duplicated across instances and the insert will fail